### ISSUE TITLE
Large integers are parsed incorrectly

### ISSUE THREAD

**Issue by rualark** [discoveryjs/discovery#53]
Example:
```
{"P":590595,"U":5678518811589061657,"A":942,"S":78,"C":1,"P":171632,"B":72057603339518945}
```

Output:
![image](https://user-images.githubusercontent.com/15198808/89177864-017c7b00-d595-11ea-976f-e8da84146e70.png)

This seems to be a high priority issue because it garbles data.

This is probably due to limitation that js stores numbers as floats. Probably this should be fixed by either custom parsing or converting to a string.

**Comment by lahmatiy** [comment]
Yeah, you hit the limit of number precision. You can see the same effect everywhere, e.g. in the console:
![image](https://user-images.githubusercontent.com/270491/89312333-71633200-d677-11ea-9480-6bdb063c3a27.png)
Possible workarounds:
- Use strings as you suggested. That's ok maybe, but you wouldn't be able to apply math operations on such numbers.
- Use BigInt. It also has it own drawbacks: JSON doesn't support for BigInt, jora doesn't support for BigInt, you cannot mix BigInt and other types, you need write such numbers in BitInt notation (i.e. `123n`)...

Custom parsing is quite complicated to implement, however it may to be added in the future. However it doesn't solve all the problems. I don't see a quick fix right now, sorry.

**Comment by rualark** [comment]
Ok. Then we can show icon near the number, that it was not parsed exactly. And add tooltip with exact (string) value.
This approximation can cause great mistakes.
